x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/write_signal_rule_exception_to_saved_object.ts

```diff
@@ -19,6 +19,9 @@ interface SignalRuleExceptionParams {
   message: string;
   services: AlertServices;
   name: string;
+  lastLookBackDate?: string | null | undefined;
+  bulkCreateTimes?: string[] | null | undefined;
+  searchAfterTimes?: string[] | null | undefined;
 }
 
 export const writeSignalRuleExceptionToSavedObject = async ({
```

Comment:
nit -> [writeCurrentStatusSucceeded](https://github.com/elastic/kibana/pull/60816/files#diff-4bac9b9a77cbb493f8af74969b7f4b68R21) looks really similar to this one

Reply:
I think these functions were split into two separate files to make it a bit easier to manage down the road. But yes they are very similar and could be merged into one function.